### PR TITLE
Create and insert new Git Machete logo

### PR DESCRIPTION
**Pull request by micpiotrowski**
Maybe the font is not the same as in git machete CLI but IMHO looks not bad ;)

WDYT about 2-colour text?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
 [![JetBrains Plugins](https://img.shields.io/jetbrains/plugin/v/14221-git-machete.svg)](https://plugins.jetbrains.com/plugin/14221-git-machete)
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/14221-git-machete.svg)](https://plugins.jetbrains.com/plugin/14221-git-machete)
 
-<img src="src/main/resources/META-INF/pluginIcon.svg" width="130" height="130" align="left"/>
+<img src="docs/logo.svg" style="width: 100%; display: block; margin-bottom: 10pt;"/>
```
Comment (PawelLipski) [inline README.md:35]: Hmm no chance for the original git font? if it's paid or sth, I can sort it out :)

Comment (PawelLipski) [reply README.md:35]: TBH I have mixed feelings about have black&red text (I'd say keep it black)... WDYT @mkondratek ?

Comment (mkondratek) [reply README.md:35]: `Paint it, black` :black_circle: we can experiment easily (but I like this first version)

actually it is a very dark brown, right? 

@micpiotrowski can you please provide here SSs with this one and other versions? ideally with some context like below

image

Comment (mkondratek) [reply README.md]: the last one wins! :trophy:

**Review by mkondratek — APPROVED** [review]
full brown text looks awesome :rocket:

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**